### DISCUSSION TITLE
Share the project to give multiple people attention

### DISCUSSION THREAD

**Discussion by Anan-GitHub** [vuesax-alpha/vuesax-alpha#54] [General]
The control is very good looking sleek and advanced, very stylish, and recently added good looking icons, I hope more front-end developers will see the participation and help the author, I shared it with my friends, thanks for the author's contribution!

**Comment by tranthinh-coding** [comment]
Thank you for liking it. But the library is still quite buggy so I haven't released the official version yet
I hope many developers know and contribute to the community with me

**Comment by Sceat** [comment]
Hey this lib is truely fantastic, I hope it become more complete with time !
